playwright/_impl/_helper.py

```diff
@@ -231,3 +231,27 @@ def make_dirs_for_file(path: Union[Path, str]) -> None:
     if not os.path.isabs(path):
         path = Path.cwd() / path
     os.makedirs(os.path.dirname(path), exist_ok=True)
+
+
+async def async_writefile(
+    file: Union[str, Path], mode: str, data: Union[str, bytes]
+) -> None:
+    def inner() -> None:
+        with open(file, mode) as fh:
+            fh.write(data)
+
+    loop = asyncio.get_running_loop()
+    await loop.run_in_executor(None, inner)
+
+
+async def async_read_binary(file: Union[str, Path]) -> bytes:
+    def inner() -> bytes:
+        with open(file, "rb") as fh:
+            return fh.read()
+
+    loop = asyncio.get_running_loop()
+    return await loop.run_in_executor(None, inner)
+
+
+async def async_read_text(file: Union[str, Path]) -> str:
+    return (await async_read_binary(file)).decode()
```

Comment:
inline this, no reason to have a separate function.

Reply:
Sure!